main/src/test/java/cgeo/geocaching/location/GeopointConverterTest.java

```diff
@@ -0,0 +1,110 @@
+package cgeo.geocaching.location;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+
+
+public class GeopointConverterTest {
+
+    private static final double DELTA = 1e-10;
+
+    private static class TestPoint {
```

Comment:
The Codacy problem here is the class name.
It contains "Test", so the check asumes that this is a class for tests.
For details see https://docs.pmd-code.org/latest/pmd_rules_java_errorprone.html#testclasswithouttestcases:

Reply:
How about renaming it to `SamplePoint` to resolve the conflict with PMD?